resources/scripts/ttx_diff.py

```diff
@@ -552,6 +566,8 @@ def main(argv):
     if not source.exists():
         sys.exit(f"No such source: {source}")
 
+    is_variable = source_is_variable(source)
```

Comment:
Since this is a function of the source and only needed in build_fontmake why not call it from there? - seems unnecessary to compute here and pass in

Reply:
good call!